### ISSUE TITLE
Move datadog.json to puppet

### ISSUE THREAD

**Issue by limed** [nubisproject/nubis-base#283]
This snippet

```bash
limed@jayne (master=):~/nubis/nubis-base/nubis/builder$ cat datadog.json
{
  "provisioners": [
  {
    "type": "file",
    "source": "nubis/bin/datadog-discover",
    "destination": "/tmp/datadog-discover",
    "order": "5"
  },
  {
    "type": "shell",
    "inline": [
      "sudo mv /tmp/datadog-discover /usr/local/bin/",
      "sudo chmod 755 /usr/local/bin/datadog-discover",
      "sudo chown root:root /usr/local/bin/datadog-discover"
    ],
    "order": "6"
  }
  ]
}
```

Instead of having packer do it, we should move it to puppet

**Comment by gozer** [comment]
+1

**Comment by limed** [comment]
Fixed in #287
